src/common/commonutils/CommonUtils.h

```diff
@@ -105,6 +106,7 @@ char* GetStringOptionFromFile(const char* fileName, const char* option, char sep
 int GetIntegerOptionFromFile(const char* fileName, const char* option, char separator, void* log);
 
 char* DuplicateString(const char* source);
+char* DuplicateStringToLowercase(const char* source);

```

Comment:
nit: I'd write as `DuplicateStringtoLowerCase`.

Reply:
"lowercase" is a word. Example: 

[https://dictionary.cambridge.org/dictionary/english/lowercase](https://dictionary.cambridge.org/dictionary/english/lowercase)